src/ReverseProxy/Forwarder/IHttpForwarderExtensions.cs

```diff
@@ -49,5 +49,29 @@ public static class IHttpForwarderExtensions
 
             return forwarder.SendAsync(context, destinationPrefix, httpClient, requestConfig, HttpTransformer.Default);
         }
+
+        /// <summary>
+        /// Forwards the incoming request to the destination server, and the response back to the client.
+        /// </summary>
+        /// <param name="context">The HttpContext to forward.</param>
+        /// <param name="destinationPrefix">The url prefix for where to forward the request to.</param>
+        /// <param name="httpClient">The HTTP client used to forward the request.</param>
+        /// <param name="requestTransform">Delegate to apply to forwarded the request.</param>
+        /// <returns>The result of forwarding the request and response.</returns>
```

Comment:
Sounds a bit strange to me since the method returns `ForwarderError` indicating if the operation succeeded or not.
```suggestion
        /// <returns>The status of a forwarding operation.</returns>
```

Reply:
This is the current summary for the existing extension methods. Should I change those?